### PR TITLE
Project 13 - Fredrik Tibbling

### PR DESCRIPTION
**Pull request by codebyfredrik**
**Link to my deployed project:** 
https://snack.expo.io/@codebyfredrik/todo-app (available through the Expo app)

**My reflections on how this week's project turned out:**
I had issues getting the the Expo CLI to work locally on my computer so I decided to do my todo app project on Expo Snack. It was nice to get some new insights about mobile development. Many things related to React are the same as in web development, but quite a few things are also totally different when it comes to the presentational layer. I also had a hard time finding the right inspiration for this project.

**Things I'd like to have clarified or explained in more detail:** 
Not for the moment.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,26 +1,81 @@
...
-      <Title>💅💅💅</Title>
-    </Container>
-  )
-}
+export default function App() {
+  const [todos, setTodos] = useState([]);
+
+  const pressHandler = key => {
+    setTodos(prevTodos => {
+      return prevTodos.filter(todo => todo.key != key);
```
Comment (JennieDalgren) [inline App.js]: use `!==` which is more strict

File: App.js [hunk 2]
```diff
@@ -1,26 +1,81 @@
...
+  };
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
+  const submitHandler = text => {
+    if (!text) return;
```
Comment (JennieDalgren) [inline App.js]: great prevention of empty submissions

File: App.js [hunk 3]
```diff
@@ -1,26 +1,81 @@
...
+          <View style={styles.list}>
+            {todos.length > 0 ? (
+              <FlatList
+                data={todos}
+                renderItem={({ item }) => (
+                  <TodoItem
+                    key={item.key}
+                    item={item}
+                    pressHandler={pressHandler}
+                  />
```
Comment (JennieDalgren) [inline App.js]: Nice work using flatlist

File: App.js [hunk 4]
```diff
@@ -1,26 +1,81 @@
...
+                )}
+              />
+            ) : (
+              <Text style={styles.notodos}>No todos to display</Text>
```
Comment (JennieDalgren) [inline App.js]: Nice with an empty state message! ⭐️

File: components/Header.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export function Header() {
+  return (
+    <View style={styles.header}>
+      <Text style={styles.title}>{'My Todos'}</Text>
```
Comment (JennieDalgren) [inline components/Header.js]: You dont need {''} around the title
`<Text style={styles.title}>My Todos</Text>`

File: App.js (hunk 1 above)
Comment (codebyfredrik) [reply App.js]: I agree.

File: components/Header.js (hunk 5 above)
Comment (codebyfredrik) [reply components/Header.js]: I don't know why I did that.

**Comment by codebyfredrik** [comment]
Thanks for great feedback! I have used styled components in the Music project (which I did a second time to correct a few things) and now in the Todo project the other week :)

**Pull request closed without merging**